main.go

```diff
@@ -272,14 +276,22 @@ func main() {
 		os.Exit(1)
 	}
 
-	clientset, clientErr := kubernetes.NewForConfig(mgr.GetConfig())
+	clientset, err := kubernetes.NewForConfig(mgr.GetConfig())
 	if err != nil {
-		setupLog.Error(clientErr, "failed to create kubernetes clientset")
+		setupLog.Error(err, "failed to create kubernetes clientset")
 	}
```

Comment:
why rename this?

Reply:
It seems my IDE did something weird here. Reverting...